FSJ_django20_project/FSJ/templates/FSJ/award.html

```diff
@@ -0,0 +1,35 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %} {# NOTE YOU MUST LOAD i18n to every HTML page even though it's also loaded to base.html #}
+
+{% block heading %}
+    {# Translators: Add award page heading #}
+    {% trans "Add Award" %}
+{% endblock %}
+
+{% block content %}
+    <div>
+        {# Translators: Greeting to the user #}        
+        {% trans "Hi" %}, {{user.first_name}} {{user.last_name}}
+    </div>
+    <br/>
+    <form action='{{ context.url }}' method='post'>
+        {% csrf_token %}
```

Comment:
The indentation of this template is all over the place with single spaces too many / too few in a bunch of places. Please tidy it up so it's easier to read the template.